src/core/notifier/event/event.go

```diff
@@ -0,0 +1,31 @@
+package event
+
+import (
+	"time"
+
+	"github.com/goharbor/harbor/src/common/models"
+	"github.com/goharbor/harbor/src/webhook/model"
+)
+
+// Event ...
+type Event struct {
```

Comment:
It seems a general event object but actually, it is not because `HookType` is for webhook, right?

Reply:
> It seems a general event object but actually, it is not because `HookType` is for webhook, right?

this struct is redundant , will delete it

Reply:
> It seems a general event object but actually, it is not because `HookType` is for webhook, right?

this struct is redundant , will delete it